### PR TITLE
[WIP] Introduce Automatic release

### PR DESCRIPTION
**Pull request by wakwak3125**
## Issue
- refs: #222 

## Overview (Required)
Details are written at #222

- [x] Introduce Firebase Crash Report plugin
- [ ] Create deploy shell.
- [ ] Create GradleTask.
- [ ] Create README

## Links

### CODE REVIEW

**Comment by konifar** [comment]
When I can review, please let me know 😉

**Comment by wakwak3125** [comment]
@konifar I got it 👍

File: .gitignore [hunk 1]
```diff
@@ -36,6 +36,7 @@ captures/
 *.iml
 .idea/*
 !.idea/codeStyleSettings.xml
+.idea/
```
Comment (jmatsu) [inline .gitignore]: You can use `git update-index --skip-worktree .idea/$FILENAME` to ignore git-controlled files with keeping changes. 😃

Comment (wakwak3125) [inline .gitignore]: @jmatsu THX! I did't know about that...👽

File: .idea/codeStyleSettings.xml [hunk 2]
```diff
@@ -1,387 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gfx) [inline .idea/codeStyleSettings.xml]: Don't remove codeStyleSettings. It is inevitable to fix code style.

Comment (wakwak3125) [inline .idea/codeStyleSettings.xml]: OK! I'll keep it 👍

**Comment by wakwak3125** [comment]
@konifar Sorry for delayed... Could you please review this PR ?
If you have any questions, suggestions, or requests feel free to ask me 👍 
I'll do my best.

**Comment by konifar** [comment]
@wakwak3125 Sure! Thanks! 👍

File: app/build.gradle [hunk 3]
```diff
@@ -4,8 +4,10 @@ apply plugin: 'jacoco'
 apply plugin: 'com.android.application'
 apply plugin: 'me.tatarka.retrolambda'
 apply plugin: 'kotlin-android'
+apply plugin: 'com.google.firebase.firebase-crash'
```
Comment (konifar) [inline app/build.gradle]: Question 🙋 
Why is this necessary?

File: README_AUTOMATED_RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+# Setup
+
+## 1. Install supply(fastlane)
+
+```
+gem install fastlane
+```
+
+## 2. Create Service Account at GooglePlayConsole.
```
Comment (konifar) [inline README_AUTOMATED_RELEASE.md]: 🙋 
Is this section almost same with fastlane setup README? If so, I think it's enough to write the link url here.
What do you think?

**Comment by konifar** [comment]
@wakwak3125 Commented some questions!

File: app/build.gradle (hunk 3 above)
Comment (wakwak3125) [inline app/build.gradle]: It can upload the mapping file to FirebaseCrash.
But, I fogot to introduce this to deploy process.
I'll fix it tomorrow.

File: README_AUTOMATED_RELEASE.md (hunk 4 above)
Comment (wakwak3125) [inline README_AUTOMATED_RELEASE.md]: Yes right.
But there are many noise to read core sentences.
So I try to rewrite about setup sentences.
But now I think I should add original link of fastlane,'s one.

**Comment by wakwak3125** [comment]
Thx!

File: README_AUTOMATED_RELEASE.md (hunk 4 above)
Comment (konifar) [inline README_AUTOMATED_RELEASE.md]: > But there are many noise to read core sentences.

Make sense.
I'm just afraid that it might be hard to follow latest setup info. But it's okay as it is! Thanks!

**Comment by horie1024** [comment]
Hi! This pull request is interesting for me😃
I think that it is better to put various script files in the `scripts` directory. What do you think?

```
scripts
    ├── bootstrap.sh
    ├── deploy2store.sh
    └── initsupply.sh
```

**Comment by konifar** [comment]
@horie1024 Cool. I agree with you.

**Comment by wakwak3125** [comment]
@horie1024 @konifar 
Thank you for your review! I applied that!
But I have one more task, so just moment please 🙏 
Sorry for delayed!

**Comment by wakwak3125** [comment]
@konifar Could you please set up Firebase Crash Report plugin?
Just download service account key and put that file to project root with file name `serviceaccount.json`.

This link will help you.
https://firebase.google.com/docs/crash/android?hl=en

**Comment by konifar** [comment]
@wakwak3125 Sure! Thanks. But I have to work another task today. Let me try tomorrow 🙇
